src/components-examples/material/datepicker/datepicker-dialog/datepicker-dialog-example.css

```diff
@@ -0,0 +1,4 @@
+.example-calendar-container {
```

Comment:
Could you avoid these styles by passing the width/height into the dialog `open` call?

Reply:
Updated